cmd/bench.go

```diff
@@ -0,0 +1,141 @@
+// Handle the "srk bench" command
+package cmd
+
+import (
+	"net"
+
+	"github.com/serverlessresearch/srk/pkg/cfbench"
+	"github.com/serverlessresearch/srk/pkg/srk"
+	"github.com/spf13/cobra"
+)
+
+// Filled in by cobra argument parsing in init()
+var benchCmdConfig struct {
+	bench        string
+	functionName string
+	functionArgs string
+	benchParams  string
+	trackingUrl  string
+	logFile      string
+}
+
+// benchCmd represents the bench command
+var benchCmd = &cobra.Command{
+	Use:   "bench",
+	Short: "Run a benchmark",
+	Long: `Run the selected benchmark. You must have already created any needed
+functions and configured the provider.`,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		provider := getProvider()
+		defer destroyProvider(provider)
+
+		var bench srk.Benchmark
+		benchArgs := srk.BenchArgs{
+			benchCmdConfig.functionName,
+			benchCmdConfig.functionArgs,
+			benchCmdConfig.benchParams,
+			benchCmdConfig.trackingUrl,
+			benchCmdConfig.bench + ".out",
+		}
+
+		switch benchCmdConfig.bench {
+		case "oneShot":
+			var err error
+			bench, err = cfbench.NewOneShot()
+			if err != nil {
+				panic("Failed to initialize OneShot benchmark")
+			}
+
+		case "concurrency_scan":
```

Comment:
Need to decide on camelCase vs under_scores

Reply:
Now that i think about it (and googling around a bit), I think dashes are pretty standard for cli args. I'll go with dashes, so "one-shot" and "concurrency-scan". Internally, we'll stick with CamelCase for names of things in code (I'm not sure if golang or others have an opinion on that, but I've been using CamelCase).